main.go

```diff
@@ -33,6 +33,8 @@ func main() {
 	dockerUser := os.Getenv("DOCKER_USER")
 	dockerPassword := os.Getenv("DOCKER_PASSWORD")
 
+	docker.InitialiseClient()
```

Comment:
It is better to parse the insecure config along with others, and pass the config to new the HTTP client.
Not sure whether `InitialiseClient` is enough readable.